### PR TITLE
Webpack dev server config

### PR DESCRIPTION
**Pull request by samelhusseini**
Webpack dev server configs need to be in the first compile.
Adjust to add test page if we are building test/

### CODE REVIEW

File: packages/webpack.config.js [hunk 1]
```diff
@@ -18,6 +18,15 @@ const packageJson = require(resolveApp('package.json'));
 console.log(`Building ${packageJson.name}`);
 
 module.exports = env => {
+    const devServer = {
+        port: 3000,
+        host: '0.0.0.0'
+    };
+    if (env.buildTest) {
+        devServer.openPage = 'test';
```
Comment (alschmiedt) [inline packages/webpack.config.js:26]: What is this part used for?

**Review by alschmiedt — APPROVED** [review]

File: packages/webpack.config.js (hunk 1 above)
Comment (samelhusseini) [reply packages/webpack.config.js:26]: openPage opens the test page if we're building a /test directory.
Or do you mean what is it used for if we aren't?

Comment (alschmiedt) [reply packages/webpack.config.js:26]: I guess I mean how does it know what test page to open.

Comment (samelhusseini) [reply packages/webpack.config.js:26]: It assumes there's an index.html. And just opens the test directory.

**Pull request merged**